pymc3/variational/advi.py

```diff
@@ -303,7 +306,10 @@ def sample_vp(
     global_RVs = list(set(model.free_RVs) - set(rvs(local_RVs)))
 
     # Make dict for replacements of random variables
-    r = MRG_RandomStreams(seed=random_seed)
+    if random_seed is None:
+        r = MRG_RandomStreams()
```

Comment:
Yes, I realized the same thing.

We probably have to consider two use-cases:
1. I don't care about the seed and want a different one each time.
2. I want reproducibility.

For 1, could we just copy current `numpy`'s seed value? For 2 we have the kwarg.